packages/signature-v4-multi-region/src/browser-build/esbuild.js

```diff
@@ -0,0 +1,20 @@
+const esbuild = require("esbuild");
```

Comment:
lets not version the bundle file, only ensure that it exists prior to running the tests, like the s3 browser bundle